src/autoscaler/db/db.go

```diff
@@ -34,3 +34,8 @@ type HistoryDB interface {
 	CanScaleApp(appId string) (bool, error)
 	Close() error
 }
+
+type ScheduleDB interface {
```

Comment:
Since we have scaling engine to synchronize active_schedule table to catch lost end, it will do delete. When schedulers sync with policy database, it will do delete too.  So looks like we don't need a dedicated pruner to remove stale active_schedule. 

There is a possibility for DB delete failure, which may cause stale data not be deleted, where a pruner will be useful. 

This is out of the scope of this PR. I will submit a fix to add timestamp in the active schedule table and use that to retrieve the latest active schedule.